content/systems/filecoin_token/payment_channels/_index.md

```diff
@@ -2,40 +2,112 @@
+	// TimeLockMax set to 0 means no timeout
+	TimeLockMax abi.ChainEpoch
+	// (optional) The SecretPreImage is used by `To` to validate
+	SecretPreimage []byte
+	// (optional) Extra can be specified by `From` to add a verification method to the voucher
+	Extra *ModVerifyParams
+	// Specifies which lane the Voucher merges into (will be created if does not exist)
+	Lane uint64
+	// Nonce is set by `From` to prevent redemption of stale vouchers on a lane
+	Nonce uint64
+	// Amount voucher can be redeemed for
+	Amount big.Int
+	// (optional) MinSettleHeight can extend channel MinSettleHeight if needed
+	MinSettleHeight abi.ChainEpoch
+
+	// (optional) Set of lanes to be merged into `Lane`
+	Merges []Merge
+
+	// Sender's signature over the voucher
+	Signature *crypto.Signature
+}
+```
+
+The client (`From`) is sending vouchers to the provider (`To`). The `Value` indicated in the voucher is progressively increasing together with the `Nonce` to indicate the value available to the provider to redeem, based on the data that the latter has sent. The provider can `Update` the balance of the channel and the balance `ToSend` to them. The provider can choose to `Collect` this balance at any time by submitting the redeemed vouchers on chain and incurring the corresponding gas cost.
+
+
+In addition, payment channels in Filecoin can be split into `lane`s created as part of updating the channel state with a payment `voucher`. Each lane has an associated `nonce` and amount of tokens it can be `redeemed` for. These lanes allow for a lot of accounting between parties to be done off-chain and reconciled via single updates to the payment channel.
+
+One example of such accounting is *merging of lanes*. When a pair of client-provider nodes have a payment channel established between them with many lanes, the provider will have to pay gas cost for each one of the lanes in order to `Collect` funds. Alternatively, the provider can send a "merge" request to the client to ask it to merge some of the lanes and consolidate the funds. As an incentive for the client to accept the merge lane request, the provider can ask for lower fees to balance out the gas cost. For example, if the provider has collected vouchers worth of 10 monetary units from two lanes, say 5 from each, and the gas cost is 2, then it can ask for 9 from the client if the latter accepts to merge the two lanes.
 
-Over the course of a transaction cycle, each party to the payment channel can send the other `voucher`s. The payment channel's  `From` account holder will send a signed voucher with a given nonce to the `To` account holder. The latter can use the voucher to `redeem` part of the lane's value, merging other lanes into it as needed.
+Over the course of a transaction cycle, each participant in the payment channel can send `voucher`s to other participants. The payment channel's client (`From`) account holder will send a signed voucher with a given nonce to the provider (`To`) account holder.
```

Comment:
I would lean away from using the terms `client` and `provider` here as they are retrieval specific. I would use `creator` (instead of client) and `recipient` (instead of provider)